File: src/main/java/estatehub/model/Review.java
package estatehub.model;

public class Review {
    private ManagerUser manager;
    private OwnerUser owner;
    private int stars;
    private String comment;

    public ManagerUser getManager() {
        return manager;
    }

    public OwnerUser getOwner() {
        return owner;
    }

    public int getStars() {
        return stars;
    }

    public String getComment() {
        return comment;
    }
}

File: model/Review.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace EstateHub.model
{
    public class Review
    {
        public string Comment { get; set; }
        public int Stars { get; set; }
        public Owner FromOwner { get; set; }
        public Manager ForManager { get; set; }
        

    }
}
